azkaban-web-server/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2012 LinkedIn Corp.
+ * Copyright 2020 LinkedIn Corp.
```

Comment:
@ypadron-in after fixing my IDE settings, this was automatically updated. Is this the right thing to do, and shall I do this for all files affected by this PR? I would suggest merging https://github.com/azkaban/azkaban/pull/2562 before this PR btw.

Reply:
I'm not an expert in the copyright topic. What I can say is that we haven't been updating the year when making a change in a file only when creating new ones. Maybe we should run this by a company lawyer to avoid surprises. 

I'll take a look at PR #2562

Reply:
All right. Note that in the [wiki](https://github.com/azkaban/azkaban/wiki/Developer-Tools-and-Tips#use-intellij-to-create-and-update-copyright-automatically) it's instructed to have IDE settings update copyright on commit.

I can revert this change to not block this PR by the legal check?

Reply:
Thank you. I'll double check the copyright matter asap and update the wiki if necessary.